README.md

```diff
@@ -768,6 +769,57 @@ expect(selectInput).not.toHaveValue(['second', 'third'])
 
 <hr />
 
+### `toHaveDisplayValue`
+
+```typescript
+toHaveDisplayValue(value: string | string[])
+```
+
+This allows you to check whether the given `<select>` element has the specified
+value.
```

Comment:
Seems this line here was not updated after you added support for other elements.

Reply:
Done, I've used the same copy of the `toHaveValue` matcher (and fixed a typo from it)